internal/service/clusteroutagesimulation/data_source_cluster_outage_simulation_test.go

```diff
@@ -66,7 +66,7 @@ func TestAccOutageSimulationClusterDS_MultiRegion_basic(t *testing.T) {
 	})
 }
 
-func testAccDataSourceMongoDBAtlasClusterOutageSimulationConfigDSSingleRegion(projectName, orgID, clusterName string) string {
+func dataSourceConfigSingleRegion(projectName, orgID, clusterName string) string {
 	return fmt.Sprintf(`
 	resource "mongodbatlas_project" "outage_project" {
 		name   = "%s"
```

Comment:
this would be better:
name   = %[1]q 

... etc.

but it's ok if you want to leave it at the moment

Reply:
Changed!